README.developer.md

```diff
@@ -1,5 +1,23 @@
+
 # Hirte
 
+- [Hirte](#hirte)
```

Comment:
I think we can omit the `Hirte` entry here.

Reply:
+1